xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2689,281 @@ rgw_override_bucket_index_max_shards = 12
    </para>
   </sect2>
  </sect1>
- <sect1 xml:id="ceph-rgw-fed">
-<!-- https://github.com/theanalyst/ceph/blob/doc/multisite-rgw/doc/radosgw/multisite.rst -->
 
+ <sect1 xml:id="ceph-rgw-fed">
   <title>Multisite &ogw;s</title>
-
+  <para>&ceph; supports several multi-site configuration options for the
+    &cogw;:</para>
   <variablelist>
    <varlistentry>
-    <term>Zone</term>
+    <term>Multi-zone</term>
     <listitem>
      <para>
-      A logical grouping of one or more &ogw; instances. There must be one zone
-      designated as the <emphasis>master</emphasis> zone in a
-      <emphasis>zonegroup</emphasis>, which handles all bucket and user
-      creation.
+      A configuration consisting of one &zonegroup; and multiple
+      zones, each zone with one or more ceph-radosgw instances. Each zone
+      is backed by its own &ceph; Storage Cluster. Multiple zones in a zone
+      group provides disaster recovery for the &zonegroup; should one of
+      the zones experience a significant failure. Each zone is
+      active and may receive write operations. In addition to disaster
+      recovery, multiple active zones may also serve as a foundation for
+      content delivery networks.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Zonegroup</term>
+    <term>Multi-zone-group</term>
     <listitem>
      <para>
-      A zonegroup consists of multiple zones. There should be a master
-      zonegroup that will handle changes to the system configuration.
+      &cogw; supports multiple &zonegroup;s, each &zonegroup; with one or
+      more zones. Objects stored to zones in one &zonegroup; within the same
+      realm as another &zonegroup; shares a global object namespace, ensuring
+      unique object IDs across &zonegroup;s and zones.
      </para>
+     <note>
+       <para>It is important to note that &zonegroup;s <emphasis>only</emphasis>
+       sync metadata amongst themselves. Data and metadata are replicated
+       between the zones within the &zonegroup;. No data or metadata is shared
+       across a realm.</para>
+     </note>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Zonegroup map</term>
+    <term>Multiple Realms</term>
     <listitem>
      <para>
-      A configuration structure that holds the map of the entire system, for
-      example which zonegroup is the master, relationships between different
-      zonegroups, and certain configuration options such as storage policies.
+      &cogw; supports the notion of realms, which
```

Comment:
Maybe rewrite the order of this, realms are namespaces, but that isn't conveyed; 
Multiple realms, which are globally unique namespaces that may encompass a single or multiple zonegroups